src/MICore/MICore.csproj

```diff
@@ -97,37 +63,33 @@
   </ItemGroup>
 
   <ItemGroup Label="Generated XML Cs files">
-    <Compile Remove="LaunchOptions.xsd.types.designer.cs" Condition="'$(IsXPlat)' == 'true'" />
-    <Compile Remove="LaunchOptions.xsd.types.designer.desktop.cs" Condition="'$(IsXPlat)' == 'false'" />
+    <Compile Remove="LaunchOptions.xsd.types.designer.desktop.cs" />
   </ItemGroup>
 
   <PropertyGroup>
-    <XsdCodeFile Condition="'$(IsXPlat)' == 'false'">LaunchOptions.xsd.types.designer.cs</XsdCodeFile>
-    <XsdCodeFile Condition="'$(IsXPlat)' == 'true'">LaunchOptions.xsd.types.designer.desktop.cs</XsdCodeFile>
+    <XsdCodeFile>LaunchOptions.xsd.types.designer.cs</XsdCodeFile>
```

Comment:
>XsdCodeFile [](http://example.com/codeflow?start=5&length=11)

I think you can remove this too

Reply:
This is still needed for `GenerateXmlSerializersAssembly.targets`